src/client.h

```diff
@@ -432,6 +433,10 @@ class Client : public con::PeerHandler, public InventoryManager, public IGameDef
 	bool sendModChannelMessage(const std::string &channel, const std::string &message);
 	ModChannel *getModChannel(const std::string &channel);
 
+	const std::string &getFormspecPrepend()
```

Comment:
const-ness can be extended to `const std::string &getFormspecPrepend() const`

Reply:
ah, I couldn't actually do this when using getEnv but I probably can now